Fitbit.Portable.Tests/AccessTokenTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Fitbit.Api.Portable;
+using Fitbit.Api.Portable.OAuth2;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Fitbit.Portable.Tests
+{
+    [TestFixture]
+    public class AccessTokenTests
+    {
+        [Test]
+        [Category("Portable")]
+        public void Can_Deserialize_AccessToken()
+        {
+            string content = SampleDataHelper.GetContent("AccessToken.json");
+
+            var deserializer = new JsonDotNetSerializer();
+            var result = deserializer.Deserialize<OAuth2AccessToken>(content);
+
+            Validate(result);
+        }
+
+        [Test]
+        [Category("Portable")]
+        public void OAuth2Helper_Can_Deserialize_AccessToken()
+        {
+            string content = SampleDataHelper.GetContent("AccessToken.json");
+
+            var result = OAuth2Helper.ParseAccessTokenResponse(content);
+
+            Validate(result);
+        }
+
+        [Test]
+        [Category("Portable")]
+        public void OAuth2Helper_Throws_FitbitTokenException()
+        {
+            string content = SampleDataHelper.GetContent("ApiError-Request-StaleToken.json");
+
+            try
+            {
+                OAuth2Helper.ParseAccessTokenResponse(content);
+            }
+            catch (FitbitTokenException exception)
+            {
+                // can only use ShouldThrow on async funcs of Task
```

Comment:
At some point we need to move all tests to async. Not for RC... higher priorities.